### PR TITLE
ErrorPage and NotResults

### PR DESCRIPTION
**Pull request by Dawid-Zieba**
Created error and not results page, repaired genres for movies

### CODE REVIEW

File: src/features/MoviesList/index.js
Comment (MateuszPiaszczyk) [inline src/features/MoviesList/index.js:1]: Looks good 👍

**Review by MateuszPiaszczyk — APPROVED** [review]
Overall, it looks good, only a few small things that would be worth improving 👍

File: src/features/PeopleList/index.js [hunk 1]
```diff
@@ -9,16 +9,20 @@ import { MainHeader } from "../../common/MainHeader";
...
     dispatch(fetchPopularPeople());
   }, []);
+  
 
   const status = useSelector(selectPopularPeopleStatus);
   const fetchResult = useSelector(selectPopularPeopleList);
-  if (status === "success") {
+  if (status === "success") 
+  {
+    if (status === "error") return <ErrorPage />
```
[the remark this answers is not in the document]
Comment (Dawid-Zieba) [reply src/features/PeopleList/index.js]: "if" is working, when i type "else" it doesn't pass and its not working ;c

[the remark this answers is not in the document]
Comment (MateuszPiaszczyk) [reply src/features/PeopleList/index.js]: oh, ok I get it then good :)

File: src/common/ErrorPage/styled.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+export const ErrorWrapper = styled.section`
+text-align: center;
+@media (max-width: ${({ theme }) => theme.breakpoint.mobileMax}px) {
+    margin-top: 140px;
+  }
+`;
+
+export const ErrorMessage = styled.p`
+font-weight: 600;
+font-size: 36;
```
[the remark this answers is not in the document]
Comment (MateuszPiaszczyk) [reply src/common/ErrorPage/styled.js]: Great! 👍

**Comment by MateuszPiaszczyk** [comment]
ok, I merge it! :)

**Pull request merged**